src/MongoDB.Driver.Core/Core/Misc/Feature.cs

```diff
@@ -35,6 +35,7 @@ public class Feature
         private static readonly Feature __aggregateGraphLookupStage = new Feature("AggregateGraphLookupStage", new SemanticVersion(3, 4, 0, "rc0"));
         private static readonly Feature __aggregateHint = new Feature("AggregateHint", new SemanticVersion(3, 6, 0, "rc0"));
         private static readonly Feature __aggregateLet = new Feature("AggregateLet", new SemanticVersion(3, 6, 0));
+        private static readonly Feature __aggregateExtendedOut = new Feature("AggregateExtendedOut", new SemanticVersion(4, 1, 6));
```

Comment:
Not sure which dev version can be specified here. The most minimum working version which I've found is `4.1.6` .